src/vm/mscorlib.h

```diff
@@ -241,19 +241,18 @@ DEFINE_FIELD_U(m_encodedArrayType, CustomAttributeType,            m_arrayType)
 DEFINE_FIELD_U(m_padding,          CustomAttributeType,            m_padding)
 
 DEFINE_CLASS_U(Globalization,          CultureInfo,        CultureInfoBaseObject)
-DEFINE_FIELD_U(compareInfo,        CultureInfoBaseObject,  compareInfo)
-DEFINE_FIELD_U(textInfo,           CultureInfoBaseObject,  textInfo)
+DEFINE_FIELD_U(_compareInfo,       CultureInfoBaseObject,  _compareInfo)
+DEFINE_FIELD_U(_textInfo,          CultureInfoBaseObject,  _textInfo)
 DEFINE_FIELD_U(numInfo,            CultureInfoBaseObject,  numInfo)
```

Comment:
Feel free to rename these two as well to have underscore for consistency.

Reply:
Will do.